keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -678,7 +679,7 @@ channel if you need any help or guidance. -->
 
 ###### Does enabling the feature change any default behavior?
 
-No
+Clients using client-go will use the aggregated discovery endpoint rather than the unaggregated discovery endpoint. This is handled automatically in client-go and clients should see less requests to the api server when fetching discovery information.
```

Comment:
Not by all versions of client-go will handle this automatically though, only 1.26 and up, right? Older versions will continue to use the old endpoints without any changes.

Reply:
Yes, thanks for pointing that out.